docs/source/cli/graph-construction/distributed/gsprocessing/aws-infra/amazon-sagemaker.rst

```diff
@@ -65,6 +65,7 @@ job, followed by the re-partitioning job, both on SageMaker:
         --job-name "${GRAPH_NAME}-${INSTANCE_COUNT}x-${INSTANCE_TYPE//./-}-${NUM_FILES}files" \
         --num-output-files ${NUM_FILES} \
         --do-repartition True \
+        --num-s3-threads 16 \
```

Comment:
This is an optional parameter, let's leave it out of the example?

Reply:
I just go through our whole doc and did not find a good place to put the doc? Or how about we put the command internally? Since not too many customers will use this argument, and it is a rare case for most customers.